pkg/helper/helpers.go

```diff
@@ -258,3 +258,13 @@ func ChunkSlice[T any](slice []T, chunkSize int) [][]T {
 func IsBYOVPC(cluster *cmv1.Cluster) bool {
 	return len(cluster.AWS().SubnetIDs()) > 0
 }
+
+func FilterEmpty(s []string) []string {
```

Comment:
NP: The naming to be more specific :pray: something like `FilterEmptyStrings` or `RemoveEmptyStrings`